source/components/JsxParser.tsx

```diff
@@ -30,6 +30,15 @@ export type TProps = {
 }
 type Scope = Record<string, any>
 
+interface ForwardRefComponentProps {
+    [key: string]: any; // for the rest of the properties
+}
+
+const ForwardRefComponent = React.forwardRef<any, ForwardRefComponentProps>((properties, ref) => {
```

Comment:
Small thing, but I'd love to have this in a separate file, and call it sometime like just ForwardRef